### PR TITLE
Fix type links and add Playwright test

### PR DESCRIPTION
**Pull request by NigelBreslaw**

### CODE REVIEW

File: docs/src/components/SlintProperty.astro [hunk 1]
```diff
@@ -41,6 +41,11 @@ if (typeName === "enum") {
 }
 
 const typeInfo = getTypeInfo(typeName);
+if (typeInfo.href !== "") {
+    const base = import.meta.env.BASE_URL;
+    typeInfo.href = `${base}${typeInfo.href}`;
+    console.log(typeInfo.href);
```
Comment (tronical) [inline docs/src/components/SlintProperty.astro:47]: Stray console.log call?

**Review by tronical — COMMENTED** [review]
I don't see the playwright bit. Did I miss it?

File: docs/src/components/SlintProperty.astro (hunk 1 above)
Comment (NigelBreslaw) [reply docs/src/components/SlintProperty.astro:47]: whoops.

**Comment by NigelBreslaw** [comment]
> I don't see the playwright bit. Did I miss it?

I moved the href links to the types into the LinkMap. Everything in the LinkMap is tested by Playwright. Should have mentioned this as its not obvious.

**Review by tronical — APPROVED** [review]

**Pull request merged**